src/Services/Air/templates/AIR_PRICE_REQ.handlebars.js

```diff
@@ -37,17 +37,31 @@ module.exports = `
                 </air:AirSegment>
                 {{/segments}}
             </air:AirItinerary>
-            {{#if platingCarrier}}
-              <air:AirPricingModifiers PlatingCarrier="{{platingCarrier}}"/>
-            {{/if}}
+            
+            <air:AirPricingModifiers
+                InventoryRequestType="DirectAccess"
+                {{#if pricing.currency}}
+                CurrencyType="{{pricing.currency}}"
+                {{/if}}
+                {{#if pricing.eTicketability}}
+                ETicketability="{{pricing.eTicketability}}"
+                {{/if}}
+                {{#if platingCarrier}}
+                PlatingCarrier="{{platingCarrier}}"
+                {{/if}}
+                {{#if pricing.faresIndicator}}
+                FaresIndicator="{{pricing.faresIndicator}}"
+                {{else}}
+                FaresIndicator="PublicAndPrivateFares"
+                {{/if}}
             {{#if business}}
-            <air:AirPricingModifiers InventoryRequestType="DirectAccess">
+            >
                 <air:PermittedCabins>
-                    <com:CabinClass Type="Business" xmlns:com="http://www.travelport.com/schema/common_v47_0" />
+                    <com:CabinClass Type="Business" />
```

Comment:
Could you please take a look at #481 and apply permitted cabins based on `permittedCabins` option?
If not, then we could create another feature request for it.

Reply:
Added cabin options similar to LFS and Availability, so we can specify Economy or First Class, with legacy "business". But to change this adequately, maybe more discussion would be required.